src/app/page/card/card.component.html

```diff
@@ -22,20 +22,20 @@ <h4>The deck is over</h4>
       <div class="card" [ngClass]="{'rotate' : routing == true}">
         <div class="front">
           <div class="header">
-              <h1 class="motto">
-                {{cards[questionNumber].question}}
-                <div *ngFor="let cardImage of cards[questionNumber].images">
-                  <img class="card-image" src="{{cardImage.image}}" alt="">
```

Comment:
alt attribute must be filled.

Reply:
Fixed. Hope that standard value is acceptable